### PR TITLE
Going all standard

### PR DESCRIPTION
**Pull request by chelm**
This PR lays out some pretty broad changes that will become the norm for all koop providers. The changes include: 

* using standard to enforce code formatting and structure and style 
* using tape and tap for testing

### CODE REVIEW

**Comment by chelm** [comment]
also introduces travis.yml to keep peeps honest

File: package.json [hunk 1]
```diff
@@ -19,9 +16,12 @@
   "author": "Chris Helm",
   "license": "BSD-2-Clause",
   "devDependencies": {
+    "koop": "~1.0.0",
+    "sinon": "^1.14.1",
+    "standard": "^3.3.0",
```
Comment (ngoldman) [inline package.json]: `"standard": "*"` is best :grin:

**Comment by chelm** [comment]
@ngoldman I wonder if the tests wont run since this adds .travis?

**Comment by ngoldman** [comment]
@chelm should be fine, generally the way I add travis support is to turn on the hook then do either a push or PR with .travis.yml -- travis reads that and tries to run the tests from there. It looks like travis is hanging as it's not even starting up the environment right now. I'd give it a little while.

**Comment by ngoldman** [comment]
@chelm looks like the first build from this PR worked -- https://travis-ci.org/koopjs/koop-socrata/builds/59428243

Travis is just extra slow today. Note tests are failing because no assertions were found though.

**Comment by chelm** [comment]
@ngoldman Looks like its failing actually. Oh I know why, probably the koop version in the dev-deps

**Comment by ngoldman** [comment]
@chelm https://travis-ci.org/koopjs/koop-socrata/jobs/59428245#L206

**Comment by chelm** [comment]
@ngoldman yar i think this last push should fix that. Was trying to use an old version of koop for testing

**Comment by jgravois** [comment]
> yar

man, i **really** hope that was an intentional 'yar'

**Comment by chelm** [comment]
@jgravois you bet it was

**Comment by ngoldman** [comment]
aye

**Comment by chelm** [comment]
@ngoldman the builds are passing, sort of 50/50 i guess. But really should be fine now

**Comment by chelm** [comment]
yup good, was the cache in travis

**Comment by ngoldman** [comment]
:+1: lgtm

**Pull request merged**